docs/components/codeblock.md

```diff
@@ -245,6 +245,19 @@ Controls whether to show a copy button which copies the original code (the value
 />
 ```
 
+### showLineNumbers
+
+Controls whether to show line numbers. Defaults to `true`.
+
+<ClientOnly>
```

Comment:
Why do we need client only here?

Reply:
I removed all `<ClientOnly>` usages from the `<KCodeBlock>` docs, and everything seems to be working fine.